README.md

```diff
@@ -31,6 +31,15 @@ Fire up the container using the provided shell script:
 $ ./test-enviroment.sh
 ```
 
+Attach to the container that was just created:
```

Comment:
These instructions are wrong now. You no longer attach to the running container, you simply run ./gradlew. The readme should not contain any instructions about docker commands, but should explain that ./gradlew will start a test container and that `./test-environment rm` removes it.